.github/workflows/trigger-dci-apps.yml

```
@@ -0,0 +1,24 @@
+# This is CI trigger for the Intel Redhat DCI-Apps Job
+# It requires a user name and a PAT to access the private repo that contains the actual action.
+# It will trigger on a pull request - can also be run manually 
+
+name: dci-apps- workflow-trigger
```

Extra space on line 5 
```suggestion
name: dci-apps-workflow-trigger
```